### DISCUSSION TITLE
What is `nx`, `ny` and `nz` values for each point in content of the `scene_dense.ply`

### DISCUSSION THREAD

**Discussion by dinushazoomi** [cdcseacave/openMVS#1038] [Q&A]
Hi all,  I am new to OpenMVS, when I read the header of `scene_dense.ply` it shows a set of values with `nx`, `ny`, and `nz`, I wonder what they represent. 

Any help would be highly appreciated
Thank you

**Comment by cdcseacave** [comment]
normal of each point

**Reply by dinushazoomi** [reply]
Sorry for my lack of knowledge on this, can you bit more explain the normal.
